### DISCUSSION TITLE
Can I add my own branch to superbench?

### DISCUSSION THREAD

**Discussion by gitaumark** [microsoft/superbenchmark#639] [Q&A]
I made some changes to the superbench code - implemented random initialization on cublaslt gemm. Is there any way I could push those changes to a branch on this repo?

**Comment by abuccts** [comment]
Hi @gitaumark, you can [fork this repo](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) under your own account (e.g., github.com/gitaumark/superbenchmark) and make modifications there.
